### DISCUSSION TITLE
Change Label from Tabs Relations

### DISCUSSION THREAD

**Discussion by kinzinho666** [filamentphp/filament#12393] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I use relation manager for 2 Models:

`public static function getRelations(): array
    {
        return [
            DocumentoCompsRelationManager::class,
            FotoCompsRelationManager::class,
        ];
    }`

How to change label names from "Documento Comps" and "Foto Comps" ?

I try change com RelationManagers/DocumentoCompsRelationManager.php like:

`class DocumentoCompsRelationManager extends RelationManager
{
    protected static string $relationship = 'documento_comps';

    //Heading page
    protected static ?string $modelLabel = 'Documento';
    protected static ?string $pluralModelLabel = 'Documentos';

    protected static ?string $icon = 'heroicon-o-document';`

but not works

![label_filament](https://github.com/filamentphp/filament/assets/29714101/4a1d7465-6410-40ab-8a36-180fc774ea85)

**Comment by dissto** [comment]
On your relation manager class you could add:

```php
protected function getTableHeading(): string|Htmlable|null
{
    return 'my title here...';
}

**Reply by kinzinho666** [reply]
This change only box 
![label_filament_2](https://github.com/filamentphp/filament/assets/29714101/ebae4a20-ffbc-48e6-be1f-0c9722ce846e)

**Reply by dissto** [reply]
```php

public static function getTitle(Model $ownerRecord, string $pageClass): string
{
    return '!!!!';
}
```

**Comment by kinzinho666** [comment]
I found solution. Thanks

protected static ?string $title = 'Documentos';
